test/unit/deploy.js

```diff
@@ -284,7 +284,8 @@ exports['Tessel.prototype.deploy'] = {
     test.expect(12);
     deployTestCode(this.tessel, {
       push: true,
-      single: false
+      single: false,
+      subargs: [],
```

Comment:
No, because this test runs code that exists at a point in which all default properties are resolved, so the `subargs` property will always be present, either empty or with contents